src/SteppedFunnelContainer.tsx

```diff
@@ -65,16 +39,47 @@ const vis: SteppedFunnelChart = {
     Object.assign(previousOptions, this.options)
 
     // add any dynamic options
-    this.options.left_axis_label = config.label_left_axis && Vizzy.stringOption("Axes", "Left Axis Label", "", 0, "label_left_axis", vis)
-    this.options.right_axis_label = config.label_right_axis && Vizzy.stringOption("Axes", "Right Axis Label", "", 0, "label_right_axis", vis)
+    this.options.left_axis_label = config.label_left_axis && Vizzy.dependString("Axes", "Left Axis Label", "", "label_left_axis", vis)
+    this.options.right_axis_label = config.label_right_axis && Vizzy.dependString("Axes", "Right Axis Label", "", "label_right_axis", vis)
 
     // register new options if options has changed since last render
     if (JSON.stringify(previousOptions) !== JSON.stringify(this.options)) {
       this.trigger && this.trigger('registerOptions', this.options)
     }
 
+    let inputRow = data[0]
+    let inputFields =  config.input_fields || queryResponse.fields.measure_like.map((f: any) => f.name)
+    inputFields !== config.input_fields && this.trigger && this.trigger("updateConfig",  [{input_fields: config.input_fields}])
+    let chunkedData: Chunk[] = inputFields.map((fieldName: string) => {
+      let datum = inputRow[fieldName]
+      return {
+        label: fieldName,
+        value: datum.value,
+        rendered: datum.rendered,
+        links: datum.links as Link[]
+      }
+    })
+    config.autosort && chunkedData.sort((a: Chunk, b: Chunk) => {
```

Comment:
I found that moving fields in the data tab didn't change order of fields in viz until after a reload. is this true?